service_layer_test.cc

```diff
@@ -221,6 +221,59 @@ TEST(ServiceLayerFunctionalityTest, ValidParentID) {
   EXPECT_EQ(sl_func.valid_parent_id("5"), false);
 }
 
+TEST(ServiceLayerFunctionalityTest, Stream) {
```

Comment:
Test comment.

Reply:
I forgot to fork the repository instead of just being on Krishna's repo but done on the forked version (new pr to come I am so sorry)